docs/general/whats-new.md

```diff
@@ -1,5 +1,28 @@
 # What's New
 
+## [v2.0.0](https://github.com/project-zot/zot/releases/tag/v2.0.0)
+
```

Comment:
"Support for [OCI Distribution Spec v1.1.0-rc3](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0-rc3) and [OCI Image Spec v1.1.0-rc4](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc4)"

Note that support for earlier release candidates is deprecated.

Reply:
^ this is THE big one

Reply:
"Built-in UI support"

Reply:
Added both items